app/components/Login.js

```diff
@@ -73,11 +74,15 @@ export const Login = () => {
       </button>
     );
   };
-
+  let redirectUri = win.location.origin + (env.electron ? '' : '/uploader');
```

Comment:
We could. This pattern was inherited directly from `blip`'s login component, which uses this sort of redefine-as-local so that it can be overridden during tests. Uploader doesn't really have UI tests, but I was planning on introducing them with the UX refresh, so I can probably just handle whatever mocks are necessary there.